qa-tests-backend/src/test/groovy/DeclarativeConfigTest.groovy

```diff
@@ -868,18 +861,16 @@ splunk:
     // verifyDeclarativeNotifier will verify that the expected auth provider exists within the API and
     // shares the same values.
     // The retrieved notifier from the API will be returned, which will have the ID field populated.
-    private Notifier verifyDeclarativeNotifier(Notifier expectedNotifier) {
+    private verifyDeclarativeNotifier(Notifier expectedNotifier) {
         def notifier = NotifierService.getNotifierClient().getNotifiers(
                 NotifierServiceOuterClass.GetNotifiersRequest
                         .newBuilder().build())
                 .notifiersList.find { it.getName() == VALID_NOTIFIER.getName() }
         assert notifier
-        assert notifier.getName() == expectedNotifier.getName()
         assert notifier.getTraits().getOrigin() == expectedNotifier.getTraits().getOrigin()
         assert notifier.getType() == "splunk"
         // Skipping the HTTP token since it will be obscured by the API.
         assert notifier.getSplunk().getHttpEndpoint() == expectedNotifier.getSplunk().getHttpEndpoint()
         assert notifier.getSplunk().getSourceTypesMap() == expectedNotifier.getSplunk().getSourceTypesMap()
```

Comment:
How about using verifiAll?

```
        assert notifier
        verifyAll(notifier) {
            name == expectedNotifier.name
            traits.origin == expectedNotifier.traits.origin
            type == 'splunk'
            splunk.httpEndpoint == expectedNotifier.splunk.httpEndpoint
            splunk.sourceTypesMap == expectedNotifier.splunk.sourceTypesMap
        }
  ```
  Why can't we compare full object?

Reply:
Hmm, not sure it'd be that easy since the ID field would be populated for the response but not for the compared object, hence I'd say its better to compare it like this.